spacy_llm/tasks/util/span.py

```diff
@@ -123,3 +124,116 @@ def _cfg_keys(self) -> List[str]:
     @property
     def _Example(self) -> Type[SpanExample]:
         return SpanExample
+
+
+class SpanTaskV2(SerializableTask[COTSpanExample]):
```

Comment:
Maybe it's a better idea to avoid defining a new class of `SpanTask` and just override the relevant methods with the new `NER` task instead?

Reply:
Hmm, actually I suppose it makes more sense this way, makes it more easily adaptable to other span tasks.

Reply:
Yes, we prefer this to generalize to other span tasks.